save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -322,7 +323,7 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
                         .decodeFromJsonString()
             setState {
                 executionDto = executionDtoFromBackend
-                filters = filters.copy(status = props.status)
+                filters = props.filters
```

Comment:
Why do we need a copy of this value in component's state? Can we use `props.filters` everywhere?